app/services/analytics.rb

```diff
@@ -20,9 +20,6 @@ def track_event(event, attributes = {})
     analytics_hash = {
       event_properties: attributes.except(:user_id),
       new_event: first_event_this_session?,
-      new_session_path: first_path_visit_this_session?,
-      new_session_success_state: first_success_state_this_session?,
-      success_state: success_state_token(event),
```

Comment:
I am a big fan of removing all of these, but just wanted to point out that we do have a dashboard that uses `success_state` [in terraform](https://github.com/18F/identity-devops/blob/66a37ec9f37934c32d63ebd26573245555abed1e/terraform/app/cloudwatch-idv-document-capture.tf#L288)

Reply:
I don't think that needs to block this PR.